.github/workflows/check.yml

```
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - staging
```

Can we just trigger the CI on every push to any branch?
Sometimes it's useful to see if the CI is still green, while working on a branch, even before opening a PR.

```suggestion
  push:
```

---

Sure